dvc/remote/ssh/connection.py

```diff
@@ -41,6 +41,14 @@ def __init__(self, host, *args, **kwargs):
         self.timeout = kwargs.get("timeout", 1800)
 
         self._ssh = paramiko.SSHClient()
+
+        # Explicitly disable paramiko logger. Due to how paramiko dynamically
+        # loads loggers, it is not disabled by DVC disable_other_loggers().
+        # See https://github.com/iterative/dvc/issues/3482
+        self._ssh.set_log_channel("dvc.paramiko")
+        logging.getLogger("dvc.paramiko").disabled = True
+        logging.getLogger("dvc.paramiko.sftp").disabled = True
```

Comment:
Can we just add a `NullHandler` to the parent (i.e. `dvc.paramiko`)? I think that makes all child loggers a no-op?

But, I'm fine as-is, it was just a question (I'm not really sure :smile:).

Reply:
@pmrowla What do you think?

Reply:
Testing it out, it works if I add a `NullHandler()` to the root `paramiko` logger, but not if we are using a custom `dvc.paramiko` log channel.

So this works
```
logging.getLogger("paramiko").addHandler(logging.NullHandler())
```

but for some reason this does not
```
self._ssh.set_log_channel("dvc.paramiko")
logging.getLogger("dvc.paramiko").addHandler(logging.NullHandler())
```

If we use `NullHandler` with the root `paramiko` logger, it also works with the statement back at the top of the file after the initial `import paramiko`, but I don't really have a preference either way.
@efiop

Reply:
@pmrowla, what about the following?
```
logging.getLogger("dvc.paramiko").addHandler(logging.NullHandler())
self._ssh.set_log_channel("dvc.paramiko")
```

Reply:
@skshetry yeah I also tried that and it does not work either

Reply:
Okay, thanks for trying out. I'll resolve this.